scripts/lib.py

```diff
@@ -172,3 +172,20 @@ def process_regression_list(testlist, test, iterations, matched_list):
         logging.info("Found matched tests: %s, iterations:%0d" %
                     (entry['test'], entry['iterations']))
         matched_list.append(entry)
+
+def check_simulator_return(output, simulator, stop_on_first_error):
+    """
+    tests simulator output for errors and terminates run if found
+    ONLY works when verbose is on (as output not returned otherwise)
+    TODO add other simulators
+    """
+
+    if not stop_on_first_error: return
+
+    if "questa32" == simulator:
```

Comment:
Should this be "questa"?
https://github.com/google/riscv-dv/blob/master/yaml/simulator.yaml#L47

Reply:
the name in our simulator.yaml is questa32 (which runs remotely on 32bit machine)  thus not the questa in your simulator.yaml - so it is the only one that I have tested to catch errors - so for us it needs to be questa32 (or coul I use an alias type of mechanism in my simulator.yaml

Reply:
I have made it 
> if "questa" in simulator:

 but have only tested on our 32 bit hosted version.